features/step_definitions/app_steps.rb

```diff
@@ -1,5 +1,6 @@
 require 'fileutils'
 require 'zip/zip'
+require 'English'
```

Comment:
To use $CHILD_STATUS instead of $?
On 31/01/2015 12:27 pm, "Adam Malcontenti-Wilson" <notifications@github.com>
wrote:

> In features/step_definitions/app_steps.rb
> <https://github.com/zendesk/zendesk_apps_tools/pull/77#discussion_r23884585>
> :
>
> > @@ -1,5 +1,6 @@
> >  require 'fileutils'
> >  require 'zip/zip'
> > +require 'English'
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/zendesk_apps_tools/pull/77/files#r23884585>.
>